src/Packetery/Module/Order/Repository.php

```diff
@@ -336,6 +346,11 @@ private function orderToDbArray( Order $order ): array {
 			$deliveryAddress = wp_json_encode( $order->getDeliveryAddress()->export() );
 		}
 
+		$apiErrorDateTime = $order->getLastApiErrorDateTime();
+		if ( null !== $apiErrorDateTime ) {
```

Comment:
Nestačila by ta podmínka jako ternár na jednu řádku ?

Reply:
mel jsem ternar primo v plneni toho $data pole, Honzovi prisel radek prilis dlouhy (viz jeho CR), opravil jsem do tohoto stavu, prijde mi to pekne citelne a kopiruje to styl o par radku vis.

Idealni (mi prijde) by bylo pouzit ?: , ale ten WP codestyle zakazuje.